packages/cli/src/commands/init/init.js

```diff
@@ -183,12 +204,14 @@ export default (async function initialize(
    */
   const version: string = minimist(process.argv).version || 'latest';
 
-  if (fs.existsSync(projectName)) {
+  const projectPath = getProjectPath({projectName, customProjectPath});
+
+  if (fs.existsSync(projectPath)) {
     throw new DirectoryAlreadyExistsError(projectName);
```

Comment:
Btw, I don't like this behavior, it's a DX regression compared to previous `init`. While we're at creation of new folders, can we prompt users if they want to overwrite the directory? You can use `inquirer` for that.

Reply:
Yeah, I've actually noticed the same. It's not strictly related to this PR, but I guess it would be a good thing to do. Old init was overwriting directories and that was quite useful generally speaking.

Reply:
This is a very nice idea, I'll do it!